architect.yml

```diff
@@ -56,12 +56,14 @@ services:
     # The liveness_probe makes sure the application is healthy - if not, it will re-start the application
     liveness_probe:
       command: curl --fail 0.0.0.0:8080
-      interval: 30s
+      interval: 10s

```

Comment:
@tjhiggins, we should keep this interval at 30s since it takes a bit of time for the the project to spin up; I still got a few probe failures on the initial startup. I'll change this, just FYI